src/Training.AirBnb.Clone.Backend/AirBnB.Persistence/Migrations/20240130111752_Add_UserCredentials.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace AirBnB.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class Add_UserCredentials : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "PasswordHash",
+                table: "Users");
+
+            migrationBuilder.CreateTable(
+                name: "UserCredentialss",
+                columns: table => new
+                {
+                    UserId = table.Column<Guid>(type: "uuid", nullable: false),
+                    PasswordHash = table.Column<string>(type: "character varying(128)", maxLength: 128, nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_UserCredentialss", x => x.UserId);
+                    table.ForeignKey(
+                        name: "FK_UserCredentialss_Users_UserId",
+                        column: x => x.UserId,
+                        principalTable: "Users",
+                        principalColumn: "Id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+        }
```

Comment:
`UserCredentials` is added as separate table here, need to re-generate migrations